wikipedia_cleanup/data_filter.py

```diff
@@ -257,6 +263,86 @@ def change_pair_needs_to_be_filtered(
         )
 
 
+class FeatureAdderFilter(AbstractDataFilter):
+    def _filter_for_property(self, changes: List[InfoboxChange]) -> List[InfoboxChange]:
+        change_timestamps = pd.Series((change.value_valid_from for change in changes))
+        change_timestamps = (
+            change_timestamps.dt.normalize().dt.normalize().dt.tz_localize(None)
+        )
+        day_of_year = change_timestamps.dt.dayofyear
+        day_of_month = change_timestamps.dt.day
+        day_of_week = change_timestamps.dt.dayofweek
+        month_of_year = change_timestamps.dt.month
+        quarter_of_year = change_timestamps.dt.quarter
+        is_month_start = change_timestamps.dt.is_month_start
+        is_month_end = change_timestamps.dt.is_month_end
+        is_quarter_start = change_timestamps.dt.is_quarter_start
+        is_quarter_end = change_timestamps.dt.is_quarter_end
+
+        days_since_last_change = (
+            change_timestamps
+            - change_timestamps.shift(+1).fillna(pd.Timestamp("20990101"))
```

Comment:
Can we have a default timestamp somewhere that explains this very high timestamp ? its reused at the bottom as well

Reply:
Added: ```DEFAULT_TIMESTAMP = pd.Timestamp("20990101")```
Though, I am not sure if you mean that this value might not be appropriate.